### PR TITLE
Task-41235 : Add MFA system step 1 : ask user a static token when he is going on a protected page

### PR DESCRIPTION
**Pull request by rdenarie**
This commit add a feature to ask a token to the user which want to access to a protected page
Protected pages are page from groups navigations configured in properties to be protected
The token is a static token for the moment, waiting for a complete integration in a next step

### CODE REVIEW

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/mfaAccess/Style.less [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+	background: @baseBackground url('@{images-path}/ShareImages/Background/texture.png') repeat top center;
+	.mfaAccessBlock {
+		width: 685px;
+		padding-top: 160px;
+		margin: 0 auto;
+		text-align: center;
+		h3 {
+			color: @baseColorMedium;
+		}
+		.mfaAccessInfo {
```
Comment (azayati) [inline platform-ui-skin/src/main/webapp/skin/less/social/portlets/mfaAccess/Style.less]: Format file

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: platform-ui-skin/src/main/webapp/skin/less/social/portlets/mfaAccess/Style.less (hunk 1 above)
Comment (rdenarie) [reply platform-ui-skin/src/main/webapp/skin/less/social/portlets/mfaAccess/Style.less]: done

**Review by azayati — DISMISSED** [review]
LGTM

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**